Automattic/pocket-casts-ios
# Watch sync improvements

| 📘 Part of: # |  
|:---:|

Fixes # 

Optimize sync process to be faster and reduce the maximum number of episodes synced on a partial refresh.

## To test

1. Start the app on the watch simulator connected to the phone using the Pocket Casts Watch scheme 
2. Add some episodes to UpNext on the phone
3. Check on the phone source on the watch if they show correctly
4. Move around the episode on the phone ( Mark as Played, Archive, Remove) 
5. Check if on the watch things show correctly

## Checklist

- [] I have considered if this change warrants user-facing release notes and have added them to `CHANGELOG.md` if necessary.
